app/task_routes.py

```diff
@@ -0,0 +1,140 @@
+from app import db
+from flask import Blueprint
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+from datetime import datetime
+# import requests
+import os
+from dotenv import load_dotenv
+from app.slack_api import notify_slack
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+load_dotenv()
+
+def validate_task(task_id):
+    try:
+        task_id = int(task_id)
+    except:
+        abort(make_response({"message": f"task {task_id} is invalid"}, 400))
+
+    task = Task.query.get(task_id)
+    
+    if not task:
+        abort(make_response({"details": f"Task {task_id} not found"}, 404))
+    
+    return task
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    
+    if "description" not in request_body or "title" not in request_body:
+        abort(make_response({"details": "Invalid data"}, 400))
+    
+    new_task = Task(title=request_body["title"],
+                    description=request_body["description"])
+    
+    
+    db.session.add(new_task)
+    db.session.commit()
+    
+    return {
+        "task": {
```

Comment:
Here you could have leveraged the `to_dict` instance method, which you did in some other route handlers below.